app/lib/l10n/app_de.arb

```diff
@@ -34,5 +34,10 @@
   "profile_page_sign_in": "Anmelden",
   "profile_page_username": "Benutzername",
 
-  "reports_page_text": "Berichte Page"
+  "reports_page_text": "Berichte Page",
+
+  "settings_page_cancel": "Abbrechen",
+  "settings_page_continue": "Fortfahren",
```

Comment:
Hmmm... maybe we should get rid of this file entirely for now (our app ownt work in german cause the texts are in english)

RIght now it is always a hastle to write everything twice in different languages and places, if we delete it we just have to add it in english but when we actually want to extend the app to another language it will be
1) unbelievably easy
2) will take considerably less time translating just the final version instead of every change